### PR TITLE
Add Czech translation + example page

### PR DESCRIPTION
**Pull request by Cu8eR**
Add Czech translation + example page

### CODE REVIEW

**Comment by Cu8eR** [comment]
#27

File: src/Backend/Modules/EloRating/Installer/Data/locale.xml [hunk 1]
```diff
@@ -5,57 +5,70 @@
       <item type="label" name="EloRating">
         <translation language="en"><![CDATA[elo-rating]]></translation>
         <translation language="nl"><![CDATA[elo-rating]]></translation>
+		<translation language="cz"><![CDATA[elo-rating]]></translation>
```
Comment (megasnort) [inline src/Backend/Modules/EloRating/Installer/Data/locale.xml]: Can you replace the tabs with spaces?

Comment (Cu8eR) [inline src/Backend/Modules/EloRating/Installer/Data/locale.xml]: Ooops, I am sorry. I will fix it ASAP

**Comment by Cu8eR** [comment]
Should be OK now

**Comment by megasnort** [comment]
Also note that in the meanwhite I added an extra label (Result) en modified one message to display the winrate (HasEloAndRanking)

**Comment by Cu8eR** [comment]
OK, now I see the issue - I use notepad++ and it has some problems with formating. I will fix this PR (+ add new lang strings) in the evening

**Comment by megasnort** [comment]
Will you create a new pull request, with the newly modified 'HasEloAndRanking'-message? It supports the win-rate data now?

**Comment by Cu8eR** [comment]
@megasnort - Yes, I will. Give me some time please, I need to pass my exam at university. Thank you!

**Comment by megasnort** [comment]
Good luck :)

**Comment by Cu8eR** [comment]
OK, last exam remains...

**Comment by Cu8eR** [comment]
@megasnort - ouch, I am really sorry. I forgot about this totaly :( is there anything what to fix?

**Comment by megasnort** [comment]
I think there was just one label missing ("result") and I added this with Google Translate and merged your work (https://github.com/megasnort/fork-cms-module-elo-rating/commit/dafe5fadb1cced0f7d9a293b787c9c042871194b)

Note that nothing changed fundamentally except the indentation of the code, and the usage of fork.settings instead of FrontendModel::getModuleSetting. Note that this requires Fork 3.9.

**Pull request merged**